src/idl_gen_kotlin.cpp

```diff
@@ -1428,6 +1442,14 @@ class KotlinGenerator : public BaseGenerator {
     }
   }
 
+  // Prepend @JvmStatic to methods in companion object.
+  static void GenerateJvmStaticAnnotation(CodeWriter &code,
+                                          bool genJvmStatic) {
+    if (genJvmStatic) {
```

Comment:
please use `snake_case`, not `camelCase`, here and elsewhere

Reply:
@aardappel  All of camel case converted to snake case.